### ISSUE TITLE
IndexError

### ISSUE THREAD

**Issue by jimcarst** [texworld/blacktex#58]
Running `blacktex 0.8.3` on `Python 3.9.12`, the following error is produced:

```py
(base) PS C:\Users> blacktex -i -c '.\in.tex'
Traceback (most recent call last):
  File "C:\Users\user\Anaconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\user\Anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\user\Anaconda3\Scripts\blacktex.exe\__main__.py", line 7, in <module>
  File "<string>", line 23, in _traverse_tree
  File "<string>", line 90, in _replace_obsolete_text_mods
IndexError: list index out of range
```

**Comment by nschloe** [comment]
Can you produce a minimal `in.tex` that shows the error?

**Comment by jimcarst** [comment]
```latex
\documentclass[]{article}
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}
\title{A}
\author{A}

\begin{document}
	
	\maketitle
	
	\begin{abstract}
		The.
	\end{abstract}
	
	\section{A}
	The the the.
\end{document}
```

Without `\fancyhf{}`, the error goes away.

**Comment by nschloe** [comment]
Fixed in 0.8.5.
